core/async/src/test_loop/mod.rs

```diff
@@ -203,7 +203,7 @@ impl TestLoopV2 {
         });
         let shutting_down = Arc::new(AtomicBool::new(false));
         // Needed for the log visualizer to know when the test loop starts.
-        tracing::info!(target: "test_loop", "TEST_LOOP_INIT");
+        tracing::info!(target: "test_loop", "test loop init");
```

Comment:
The TEST_LOOP_INIT is a special string that is used for some other automation, please revert.

Reply:
Yeah, makes sense, reverted. Ideally log format shouldn't have external dependencies tho.